items.json

```diff
@@ -22756,9 +22756,9 @@
     "name": "Ridged Great Saber",
     "culture": "Neutral",
     "type": "TwoHandedWeapon",
-    "price": 8001,
+    "price": 50,

```

Comment:
why?

Reply:
not sure what that is, did not intend to change it. Sorry for the mistake